src/js/view/view.js

```diff
@@ -856,6 +856,43 @@ function View(_api, _model) {
         return _model.get('isFloating') ? _wrapperElement : _playerElement;
     }
 
+    function _createFloatingDragUI() {
```

Comment:
Make a new module in view for this. A class that accepts _wrapperElement as the `element` argument would make sense.

`this.floatingUI = new FloatingDragUI(_wrapperElement);`